src/components/Editor/index.js

```diff
@@ -188,11 +197,14 @@ class Editor extends Component {
   }
 
   render() {
-    const { contentEditable, className, style } = this.props
+    const { contentEditable, className, style, ...rest } = this.props
     const { html } = this.state
+    delete rest.children;
```

Comment:
Interesting.  I don't think the lifetime of `rest` is such that it will be affected by the V8 issue, but I shall make the change requested and leave children alone.